app/components/register/register.component.js

```diff
@@ -15,13 +16,15 @@ const RegistrationForm = () => (
       <Segment stacked>
         <input type="hidden" name="response_type" value="code" />
         <input type="hidden" name="client_id" value="228MZ3" />
-        <input type="hidden" name="redirect_uri" value="https://capcoglobalchallenge.com/fitbit/auth" />
+        {/*}<input type="hidden" name="redirect_uri" value="https://capcoglobalchallenge.com/fitbit/auth" />*/}
+        <input type="hidden" name="redirect_uri" value="https://localhost/fitbit/auth" />
```

Comment:
Was this changed for testing locally?

Reply:
Yes. I'm tempted to say leave that as localhost for now as we don't know what the new URL will be